scripts/benchmark_inference.py

```diff
@@ -132,6 +132,11 @@
 parser.add_argument(
     "--skip_e2e_runs", action="store_true", help="Do not run the e2e benchmarks"
 )
+parser.add_argument(
+    "--unfuse_kernels",
```

Comment:
nit: might be better to say 'weight's or 'kqv' instead of 'kernels' since kernel fusion would typically refer to compilation and operator fusion